python/tskit/drawing.py

```diff
@@ -857,6 +886,42 @@ def __init__(
                 f"Mutations {unplotted} are above nodes which are not present in the "
                 "displayed tree, so are not plotted on the topology."
             )
+        self.left_extent = tree.interval.left
+        self.right_extent = tree.interval.right
+        if all_edge_mutations:
+            tree_left = tree.interval.left
+            tree_right = tree.interval.right
+            edge_left = ts.tables.edges.left
+            edge_right = ts.tables.edges.right
+            node_edges = tree._node_edges()
+            # whittle mutations down so we only need look at those above the tree nodes
+            mut_t = ts.tables.mutations
+            focal_mutations = np.isin(mut_t.node, np.fromiter(nodes, mut_t.node.dtype))
+            mutation_nodes = mut_t.node[focal_mutations]
+            mutation_positions = ts.tables.sites.position[mut_t.site][focal_mutations]
+            mutation_ids = np.arange(ts.num_mutations, dtype=int)[focal_mutations]
+            for m_id, node, pos in zip(
+                mutation_ids, mutation_nodes, mutation_positions
+            ):
+                curr_edge = node_edges[node]
+                if curr_edge >= 0:
```

Comment:
Yep. I tend to use `>=0` because it tells me on re-reading that we are testing for a positive (i.e. valid) curr_edge index. But since we never fill with values other than 0..num_edges or tskit.NULL, it's the same. Happy to switch to ` != tskit.NULL` if you think it's more readable.